sakuya/wordle.py

```diff
@@ -122,6 +123,10 @@ async def guess(self, ctx: commands.Context, guess: str):
             await ctx.send("Your guess must be 5 letters, a-z only.")
             return
         guess = guess.lower()
+        if state.game_start != current_game_start() and state.in_progress:
+            state.in_progress = False
```

Comment:
As for the issue of only sending the message once, you can use the existing reset method `self.reset(state.guild)` to accomplish the same without additional state – though it should probably be moved to after the solution is accessed then.